entity-framework/core/get-started/aspnetcore/new-db.md

```diff
@@ -1,130 +1,128 @@
 
-[!code-csharp[Main](../../../../samples/core/GetStarted/AspNetCore/EFGetStarted.AspNetCore.NewDb/Startup.cs?name=ConfigureServices&highlight=7-8)]
+* Add the following code to the `ConfigureServices` method:
+ [!code-csharp[Main](../../../../samples/core/GetStarted/AspNetCore/EFGetStarted.AspNetCore.NewDb/Startup.cs?name=ConfigureServices&highlight=7-8)]
 
-> [!NOTE]
-> In a real application you would typically put the connection string in a configuration file. For the sake of simplicity, we are defining it in code. For more information, see [Connection Strings](../../miscellaneous/connection-strings.md).
+Note: A real app would gennerally put the connection string in a configuration file. For the sake of simplicity, we are defining it in code. See [Connection Strings](../../miscellaneous/connection-strings.md) for more information.
 
 ## Create your database
 
-Now that you have a model, you can use migrations to create a database for you.
+Once you have a model, you can use [migrations](https://docs.microsoft.com/aspnet/core/data/ef-mvc/migrations#introduction-to-migrations) to create a database.
+
+* Open the PMC:
+
+  **Tools –> NuGet Package Manager –> Package Manager Console**
+* Run `Add-Migration InitialCreate` to scaffold a migration to create the initial set of tables for your model. If you receive an error stating `The term 'add-migration' is not recognized as the name of a cmdlet`, close and reopen Visual Studio.
+* Run `Update-Database` to apply the new migration to the database. This command creates the database before applying migrations. 
 
-* **Tools –> NuGet Package Manager –> Package Manager Console**
-* Run `Add-Migration MyFirstMigration` to scaffold a migration to create the initial set of tables for your model. If you receive an error stating `The term 'add-migration' is not recognized as the name of a cmdlet`, then close and reopen Visual Studio
-* Run `Update-Database` to apply the new migration to the database. Because your database doesn't exist yet, it will be created for you before the migration is applied.
+Notes: 
 
-> [!TIP]
-> If you make future changes to your model, you can use the `Add-Migration` command to scaffold a new migration to make the corresponding schema changes to the database. Once you have checked the scaffolded code (and made any required changes), you can use the `Update-Database` command to apply the changes to the database.
->
->EF uses a `__EFMigrationsHistory` table in the database to keep track of which migrations have already been applied to the database.
+* If you make future changes to your model, you can use the `Add-Migration` command to scaffold a new migration to make the corresponding schema changes to the database. See [Migrations](https://docs.microsoft.com/aspnet/core/data/ef-mvc/migrations#introduction-to-migrations) for more information.
+* EF uses a `__EFMigrationsHistory` table in the database to keep track of which migrations have already been applied to the database.
```

Comment:
Great idea. I was being conservative with the removal. Removing them will make the tutorials smoother and easier to maintain.